packages/sui-test-contract/src/setup/utils.js

```diff
@@ -61,7 +61,7 @@ export const writerFactory = providers => (path, data) => {
   })
 
   console.log(`Writing the Pact file "${path}"`) // eslint-disable-line
-  writeData2File(path, data)
+  createWriter()(path, data)
```

Comment:
Fix for a breaking change introduced in [v1.4.0](https://github.com/pactflow/pact-msw-adapter/releases/tag/v1.4.0): https://github.com/pactflow/pact-msw-adapter/commit/6d4554158d4ef2671e88e556604319379659b5a2